_includes/county-map.html

```diff
@@ -42,4 +56,28 @@
       <use xlink:href="{{ states_svg }}#state-{{ include.state }}"></use>
     </g>
   </svg>
+
+  {% if include.toggle %}
+    <h4 class="details-container">
+      <button is="aria-toggle"
+        aria-controls="{{ include.toggle }}">&plus; Details</button>
+    </h4>
+  {% endif %}
+
+
 </div>
+
+<dl class="legend-container">
```

Comment:
hmm. Based on the formatting, I think I need nesting, so I'll try a list approach